src/main/java/gg/essential/universal/UGraphics.java

```diff
@@ -416,7 +416,7 @@ public static void drawString(UMatrixStack stack, String text, float x, float y,
         //$$ UMinecraft.getFontRenderer().drawString(text, x + 1f, y + 1f, shadowColor);
         //$$ UMinecraft.getFontRenderer().drawString(text, x, y, color);
         //#else
-        UMinecraft.getFontRenderer().drawString(text, x + 1f, y + 1f, shadowColor, false);
+        UMinecraft.getFontRenderer().drawString(ChatColor.Companion.stripColorCodes(text), x + 1f, y + 1f, shadowColor, false);
```

Comment:
This line applies to 1.12.2 and below only, there's another one for 1.15 and another one for 1.16+.
(Please store the shadow text in a local, DRY)